artipie-main/Dockerfile

```diff
@@ -15,8 +14,10 @@ USER 2021:2020
 COPY target/dependency  /usr/lib/artipie/lib
 COPY target/${JAR_FILE} /usr/lib/artipie/artipie.jar
 
+RUN timeout 10s java -XX:ArchiveClassesAtExit=/usr/lib/artipie/artipie-d.jsa $JVM_ARGS --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.security=ALL-UNNAMED -cp /usr/lib/artipie/artipie.jar:/usr/lib/artipie/lib/* com.artipie.VertxMain --config-file=/etc/artipie/artipie.yml --port=8080 --api-port=8086 || :
```

Comment:
What it's for?

Reply:
@dgarus , this line runs Artipie server for 10sec. to save loaded classes in JVM AppCDS cache, which will be used to speed-up startup of the container (see changes in that file later). 
Useful links:
https://docs.oracle.com/en/java/javase/11/vm/class-data-sharing.html#GUID-EC975B2E-B4AB-45B4-B91F-51C3A264D0CE
https://www.morling.dev/blog/building-class-data-sharing-archives-with-apache-maven/